.github/workflows/prevent-msgid.yml

```diff
@@ -0,0 +1,41 @@
+name: Prevent msgid changes in translations
+
+on:
+  pull_request:
+    paths:
+      - "**/*.po"
+
+jobs:
+  prevent-msgid-changes:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Check for changes between msgid and msgstr
+        run: |
+          git reset origin/main
+          for filename in $(git diff --exit-code --name-only | grep '.po')
```

Comment:
Small question: I wonder why `--exit-code` is here? The man page says

> Make the program exit with codes similar to diff(1). That is, it exits with 1 if there were differences and 0 means no differences.

but it's not clear to me why we want the exit code here?